components/SignUpForm.tsx

```diff
@@ -1,287 +1,128 @@
+              type='text'
+              value={handle}
+              placeholder='IG/Twitter Handle'
+              onChange={(event) => setHandle(event.currentTarget.value)}
+              name='handle'
+              id='handle'
+            />
           </div>
-        </>
-      )}
-      {selected === 3 && (
-        <div className='row'>
-          <div className='description'>
-            <p>
-              Your Submission been received!
-              <br />
-              <br />
-              <strong>
-                Expect a text with instructions to complete your wager via CASH
-                APP. Your entry is not valid until we receive confirmation of
-                your wager via Cash App.
-              </strong>
-              <br />
-              <br />
-              All pay outs will be dispersed through Cash App as well. Winnings
-              can be used for future gameplay.
-              <br />
-              <br />
-              If you did not receive a text, you can click below to pay as well
-              <br />
-              <br />
-              <a href={`https://cash.app/$Cloutyio/${wager}`}>
-                <img className='cashapp' src='/static/img/cashapp.png' />
-              </a>{' '}
-              <br />
-              <br />
-              The final entry date is <strong>Tuesday, December 3rd.</strong>
-              <br />
-              <br />
-              <br />
-              <i>
-                For all issues please email{' '}
-                <a href='mailto: payments@clouty.io'> payments@clouty.io</a>.
-                For all media inquiries please email
-                <a href='mailto: breemz@clouty.io'> breemz@clouty.io</a>
-              </i>
-            </p>
+          <br />
+          <div>
+            <label htmlFor='email'>Select your wager:</label>
+            <Radio.Group
+              onChange={(event: RadioChangeEvent) =>
+                setWager(Number(event.target.value))
+              }>
+              <Radio className='radio' value={1}>
+                $1
+              </Radio>
+              <br />
+              <Radio className='radio' value={5}>
+                $5
+              </Radio>
+              <br />
+              <Radio className='radio' value={10}>
+                $10
+              </Radio>
+            </Radio.Group>
+          </div>
+          <br />
+          <div>
+            <Cleave
+              className='ant-input'
+              placeholder='Phone Number'
+              onChange={(event) => setPhoneNumber(event.currentTarget.value)}
+              options={{ phone: true, phoneRegionCode: 'US' }}
+            />
           </div>
-        </div>
-      )}
-    </>
+          <Question
+            type={game.type}
+            options={game.options}
+            title={game.title}
+            setAnswer={setAnswer}
+          />
+          <br />
+          <br />
+          {phoneNumber && answer && wager && name && email && (
```

Comment:
this isn't the best way to validate input. for example, email should only be valid with `@`.

Reply:
I agreee